atlas/fixtures/basic/@.gql

```diff
@@ -0,0 +1,9 @@
+schema @id(name: "lyrical") {
+  query: Query
+  mutation: Mutation
+}
+
+type Query {
+  findUser(byId: ID): User @from(field: "users.Query.findUser")
+  me: User @from(field: "auth.Query.me")
```

Comment:
with the other file being named `admin`, how does atlas know to call this `auth`? Does each service need to name its graph with an id?